arch/arc/core/arc_smp.c

```diff
@@ -103,18 +72,60 @@ void z_arch_slave_start(int cpu_num)
 {
```

Comment:
How `slave_start()` could be executed w/o SMP at all?

Reply:
see tests/kernel/spinlock and tests/kernel/mp

Reply:
Well only in `tests/kernel/mp`! In `tests/kernel/spinlock` it's only referred in comments.